src/services/createRouterFind.spec.ts

```diff
@@ -41,7 +41,8 @@ test('when given a url that does not match a route returns undefined', () => {
   expect(route).toBeUndefined()
 })
 
-test('when given an external url that does not match a route returns undefined', () => {
+// until route has host this test is not useful
+test.fails('when given an external url that does not match a route returns undefined', () => {
```

Comment:
This doesn't seem right. Because router.push supports any url right?

Reply:
okay, reverted back to stripping host from initialUrl then